app.js

```diff
@@ -0,0 +1,18 @@
+const navBarItems = document.querySelector('.nav-bar').children;
+const menuClose = document.querySelector('.close-hamb');
+// array distructuring
+const [navBarName, navList, menuOpen, containerNav] = navBarItems;
+// console.log('object', navBarName, navBarLink, navBarHumb, navBarContainer);
+
+menuOpen.addEventListener('click', () => {
+  containerNav.style.display = 'block';
+  navList.style.display = 'none';
+  menuOpen.style.display = 'none';
+  navBarName.style.display = 'none';
+});
+
+menuClose.addEventListener('click', () => {
```

Comment:
- [ ] [OPTIONAL] For semantic reasons kindly make the cursor effect of the close button a pointer, as you've done to the hamburger menu.